iRecruit-express-app/routes/users.js

```diff
@@ -40,6 +56,14 @@ router.post('/users', async (req, res) => {
 
 // Route for user login
 router.post('/users/login', async (req, res) => {
+  // Check if data is in cache
+  const cachedData = memcache.get('users:login');
```

Comment:
We should never cache login pages, this is just letting anyone see the previous user's logged in state.

Reply:
I misunderstood caching and had not grasped the main idea then, but thank you, I got it now.